helix-term/src/commands.rs

```diff
@@ -1844,7 +1844,10 @@ fn global_search(cx: &mut Context) {
                                 entry.path(),
                                 sinks::UTF8(|line_num, _| {
                                     all_matches_sx
-                                        .send(FileResult::new(entry.path(), line_num as usize - 1))
+                                        .send(FileResult::new(
+                                            entry.path(),
+                                            (line_num as usize).saturating_sub(1),
```

Comment:
I think line_num is guaranteed to be non-zero here too

Reply:
If this is also guaranteed to be non-zero here, I close this pull request.